guardian_linter_watchdog.py

```
@@ -21,43 +20,58 @@
 
-logging.basicConfig(level=logging.INFO, format='%(asctime)s - %(levelname)s - %(message)s')
+logging.basicConfig(
+    level=logging.INFO, format="%(asctime)s - %(levelname)s - %(message)s"
+)
 logger = logging.getLogger(__name__)
 
+
 async def run_linter(file_path: Path):
     """Run the linter on a specific file."""
     if not any(part in EXCLUDED_DIRS for part in file_path.parts):
         command = LINT_COMMAND + [str(file_path)]
```

Ruff (0.11.9)

Consider `[*LINT_COMMAND, str(file_path)]` instead of concatenation

Replace with `[*LINT_COMMAND, str(file_path)]`

(RUF005)